networkx/classes/graph.py

```diff
@@ -1100,7 +1100,7 @@ def update(self, edges=None, nodes=None):
         It you want to update the graph using an adjacency structure
         it is straightforward to obtain the edges/nodes from adjacency.
         The following examples provide common cases, your adjacency may
-        be slightly different and require tweaks of these examples.
+        be slightly different and require tweaks of these examples::
```

Comment:
I don't think the double colons are necessary within the `Examples` section of the docstring as numpydoc will parse the doctest-formatted lines regardless of indentation.

Reply:
The trouble here seems to be the “notes” section which interrupts the examples section. I think I need the double colon to keep the namespace from the examples section alive for the doc tests in the notes section.